spec/services/doc_auth/error_generator_spec.rb

```diff
@@ -550,7 +550,7 @@ def build_error_info(
         it 'DocAuthResult is failed with selfie error' do
           error_info = build_error_info(doc_result: 'Passed', image_metrics: metrics)
           errors = described_class.new(config).generate_doc_auth_errors(error_info)
-          expect(errors.keys).to contain_exactly(:general, :selfie, :hints)
+          expect(errors.keys).to contain_exactly(:front, :back, :general, :selfie, :hints)
```

Comment:
Based on the comments on L547 and L549, I don't think this test is finished?

Reply:
Ah, thanks for catching that. Those comments are actually out of date. I removed them [here](https://github.com/18F/identity-idp/pull/10042/commits/6d90605756888e7c1270411e03bf82b2f6294a2e).

I verified that it's the comments that are out of date by changing `Pass` to `Fail` and vice versa. That causes the tests to fail (which is what we want). I think that we actually fixed this in past mock proofer work.